src/zed_wrapper_node.cpp

```diff
@@ -298,7 +299,7 @@ int main(int argc, char **argv) {
     nh_ns.getParam("cloud_frame_id", cloud_frame_id);
 
     // Create the ZED object
-    sl::zed::Camera *zed;
+    std::unique_ptr<sl::zed::Camera> zed;
     if (argc == 2) {
         zed = new sl::zed::Camera(argv[1]); // Argument "svo_file" in launch file
```

Comment:
What error does it give? I'm afraid I don't have access to a suitable compiler on my platform, and access to my embedded target is intermittent